<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_16">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tables">
       <attribute name="title">
        <string>Table Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>7</number>
          </property>
          <widget class="QWidget" name="pops">
           <attribute name="title">
            <string>Pops</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QGroupBox" name="popsJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <widget class="QPushButton" name="popsSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="popsLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="popsTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="races">
           <attribute name="title">
            <string>Races</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <widget class="QGroupBox" name="racesJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <widget class="QPushButton" name="racesSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="racesLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="racesTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="religions">
           <attribute name="title">
            <string>Religions</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QGroupBox" name="religionsJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QPushButton" name="religionsSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="reigionsLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="religionsTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="agegroups">
           <attribute name="title">
            <string>Age Groups</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QGroupBox" name="agegroupsJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="ageGroupsSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="ageGroupsLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="agegroupsTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="occupations">
           <attribute name="title">
            <string>Occupations</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <widget class="QGroupBox" name="occupationsJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_17">
               <item row="0" column="0">
                <widget class="QPushButton" name="occupationsSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="occupationsLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="occupationsTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="countries">
           <attribute name="title">
            <string>Countries</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QGroupBox" name="countriesJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QPushButton" name="countriesSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="countriesLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="countriesTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="regions">
           <attribute name="title">
            <string>Regions</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QGroupBox" name="regionsJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QPushButton" name="regionsSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="regionsLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="regionsTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="areas">
           <attribute name="title">
            <string>Areas</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QGroupBox" name="areasJSON">
              <property name="title">
               <string>Save to / load from JSON</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QPushButton" name="areasSave2JSON">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="areasLoadFromJSON">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="areasTable"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="random">
       <attribute name="title">
        <string>Randomizer</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
